php/libraries/TimePoint.class.inc

```diff
@@ -208,7 +209,8 @@ class TimePoint
         int $SubprojectID,
         ?string $nextVisitLabel,
         \Site $psc,
-        \Project $project
+        \Project $project,
+        int $lang = 1
```

Comment:
Nothing garantees you that language id `1` exists !!

Reply:
But en-CA will. 


> 
> I specially like the fact that you do not use `language_id` which is meaningless anyway.

Reply:
So the reason I went away from using language_code is as follows:
1) The utility getLanguageList returns an array with `[language_id] => language_label`
2) When preferred language was implemented for users, language ID is also used. 

It just seems we should keep that convention going?

Reply:
kill IDs